src/components/resumes/ResumesAPI.js

```diff
@@ -6,36 +6,62 @@ const API_VERSION = "v1";
 
 export async function postResume(resume) {
   try {
-    const resumes = await axios.post(
+    const result = await axios.post(
       "http://localhost:7071/api/resumes",
```

Comment:
can you update this to follow our new pattern. 
![image](https://user-images.githubusercontent.com/85913826/127956234-898ffa15-4fe6-4ee0-b84c-d8386dbf018f.png)